src/sentry/static/sentry/app/components/events/interfaces/spans/header.tsx

```diff
@@ -551,20 +543,14 @@ const TickLabel = (props: {
 
 const DurationGuideBox = styled('div')<{alignLeft: boolean}>`
   position: absolute;
-
   background-color: ${p => p.theme.white};
   padding: 4px;
-
   border-radius: 3px;
   border: 1px solid rgba(0, 0, 0, 0.1);
-
   height: 16px;
-
   line-height: 1;
   vertical-align: middle;
-
   transform: translateY(50%);
```

Comment:
@doralchan This likely needs adjusting now that the duration guide box extends slightly outside the time axis component. 
<img width="1029" alt="Screen Shot 2019-12-03 at 5 11 12 AM" src="https://user-images.githubusercontent.com/139499/70041559-6194ca80-158b-11ea-9970-cc68ea90faf6.png">

Would you be interested in looking into this?

Reply:
Whoa - how do I view this?

Reply:
@doralchan  if you hover your mouse hover the minimap, you can see the duration box guide

Reply:
![Screen Shot 2019-12-03 at 9 22 06 AM](https://user-images.githubusercontent.com/4830259/70073932-a67e2880-15ae-11ea-96b8-a874c497236c.png)
✔️ Okie doke